ipwb/check_for_update.py

```diff
@@ -0,0 +1,30 @@
+import logging
+import re
+
+import requests
+
+from ipwb import __version__ as ipwb_version
+
+logger = logging.getLogger(__name__)
+
+
+def get_latest_ipwb_version() -> str:
+    """Fetch latest IPWB version from PyPI."""
+    response = requests.get('https://pypi.python.org/pypi/ipwb/json').json()
```

Comment:
This may raise an exception if the PyPI service is not healthy, which should be handled.

Reply:
I will add a bit of user friendliness.

Reply:
Eventually we may want to decouple from pypi (e.g., binaries on IPFS) but moving this magic URI string out of the code might help with code quality, albeit, this should be the only place we use it.